platform/darwin/src/MGLMapSnapshotter.h

```diff
@@ -12,40 +12,46 @@ MGL_EXPORT
 @interface MGLMapSnapshotOptions : NSObject
 
 /**
- Creates a set of options with the minimum required information
- @param styleURL the style url to use
- @param camera the camera settings
- @param size the image size
+ Creates a set of options with the minimum required information.
+ 
+ @param styleURL URL of the map style to snapshot. The URL may be a full HTTP or HTTPS URL,
+ a Mapbox URL indicating the style’s map ID (mapbox://styles/{user}/{style}), or a path
+ to a local file relative to the application’s resource path. Specify nil for the default style.
+ @param size The image size.
  */
-- (instancetype)initWithStyleURL:(NSURL*)styleURL camera:(MGLMapCamera*)camera size:(CGSize)size;
+- (instancetype)initWithStyleURL:(nullable NSURL*)styleURL camera:(MGLMapCamera*)camera size:(CGSize)size;
 
-#pragma mark - Configuring the map
+#pragma mark - Configuring the Map
 
 /**
-  The style URL for these options.
+ URL of the map style to snapshot. The URL may be a full HTTP or HTTPS URL,
+ a Mapbox URL indicating the style’s map ID (mapbox://styles/{user}/{style}), or a path
+ to a local file relative to the application’s resource path. Specify nil for the default style.
  */
-@property (nonatomic, readonly) NSURL* styleURL;
+@property (nonatomic, readonly) NSURL *styleURL;
 
 /**
- The zoom. Default is 0.
+ The default zoom level is 0. Overrides the altitude in the mapCamera options if set.
  */
-@property (nonatomic) double zoom;
+@property (nonatomic) double zoomLevel;
 
 /**
- The `MGLMapcamera` options to use.
+ A camera representing the viewport visible in the snapshot.
  */
-@property (nonatomic) MGLMapCamera* camera;
+@property (nonatomic) MGLMapCamera *camera;
 
 /**
- A region to capture. Overrides the center coordinate
- in the mapCamera options if set
+ The cooordinate rectangle that encompasses the bounds to capture. Overrides the center coordinate
+ in the mapCamera options if set.
  */
-@property (nonatomic) MGLCoordinateBounds region;
+@property (nonatomic) MGLCoordinateBounds coordinateBounds;
 
-#pragma mark - Configuring the image
+#pragma mark - Configuring the Image
 
 /**
- The size of the output image. Minimum is 64x64
+ The size of the output image.
+ 
+ The image may be no less than 64 pixels wide and no less than 64 pixels tall.
```

Comment:
OK, let’s clarify that we really mean pixels here, because most developers will read this thinking it’s a typo:

> The image may be no less than 64 pixels wide (32 points wide on a 2× display) and no less than 64 pixels tall.

Reply:
Ultimately we should handle the conversion ourselves so that developers don’t have to multiply by the backing scale factor, which is inconvenient and inconsistent with MapKit: #10088.

Reply:
Internally we handle the scale factor so size is actually points instead of pixels in the end. Do you think this clarifies the issue?

> The size of the output image may be no less than the following dimensions:
>> Non-retina 64 points wide and tall.
>> 2× 32 points wide and tall.
>> 3× 22 points wide and tall.